src/common/util.cc

```diff
@@ -164,6 +164,10 @@ void collect_sys_info(map<string, string> *m, CephContext *cct)
     (*m)["container_name"] = container_name;
     in_container = true;
   }
+  if (const char *container_image = getenv("CONTAINER_IMAGE")) {
```

Comment:
Following Rook's convention, I guess it'll either be `POD_CONTAINER_IMAGE` or `ROOK_CONTAINER_IMAGE`. @travisn wdyt?

However, we can make this really generic so that it'll work for any containerized deployment (not k8s specific). In that case, `CONTAINER_IMAGE` is fine by me.

Reply:
Rook does create several generic env vars, so this is consistent. For example, `POD_NAME`, `POD_NAMESPACE`, and `NODE_NAME` are set on the pod spec by rook.

Reply:
the question is, do Rook and generic containers both set this exact variable today, no?

Reply:
Rook will set it with this PR: https://github.com/rook/rook/pull/2757, and other environments would ideally set it as well.

Reply:
@dsavineau please add it to ceph-ansible in all the daemons systemd units. Thanks.